mods/ts/installer/firestorm.yaml

```
@@ -4,6 +4,7 @@ fstorm: Firestorm Expansion Disc (English)
 		Install/Language.dll: 4df87c1a2289da57dd14d0a7299546f37357fcca
 	Install:
 		copy: .
+			^SupportDir|Content/ts/firestorm/scores01.mix: Install/scores01.mix
```

The `scores01.mix` is actually not in the `Install` directory where `expand01.mix` is.  🤦‍♂️ 
At least not on the ISO I downloaded from  https://cnc-comm.com/tiberian-sun/downloads/the-game/firestorm-disc
```suggestion
			^SupportDir|Content/ts/firestorm/scores01.mix: scores01.mix
```

---

That's because the original games in their disc form never installed music to disk, just like movies they always read it from CD.